### PR TITLE
Removed code duplication from Mobile.cs

### PR DESCRIPTION
**Pull request by Mailaender**
Detected by dupFinder.

### CODE REVIEW

File: OpenRA.Mods.Common/Traits/Mobile.cs [hunk 1]
```diff
@@ -221,32 +221,7 @@ public bool CanEnterCell(World world, Actor self, CPos cell, out int movementCos
...
-						return false;
-					var crushables = a.TraitsImplementing<ICrushable>();
-					if (!crushables.Any())
-						return false;
-					foreach (var crushable in crushables)
-						if (!crushable.CrushableBy(Crushes, self.Owner))
-							return false;
-				}
-			}
+			CanMoveFreelyInto(world, self, cell, ignoreActor, check);
```
Comment (chrisforbes) [inline OpenRA.Mods.Common/Traits/Mobile.cs]: Do you mean `return CanMoveFreelyInto(...` ?

As is, this isn't equivalent at all.

Comment (Mailaender) [inline OpenRA.Mods.Common/Traits/Mobile.cs]: Yes, fixed. Thanks.

**Comment by abcdefg30** [comment]
Looks good to me. :+1: / :white_check_mark:

**Comment by obrakmann** [comment]
Looks fine, stuff still moves. :+1:

**Pull request merged**